### ISSUE TITLE
Add a function to take a screenshot

### ISSUE THREAD

**Issue by vincentducorps** [Freeeaky/GTALua#64]
Something like this will be awesome !

``` lua
graphics.TakeScreenshot(str file_name, str folder_name, bool hide_hud)
```

Screenshot is placed at the root of the addon unless a folder name is specified.

Significant time savings to fill the wiki ;)

**Comment by Freeeaky** [comment]
That's a very good idea, but it's a lot of work. I'll have to support both DirectX versions (9 & 11), do image conversions and stuff.

(Btw: Working on the other issues right now, fixing the native inis)

**Comment by vincentducorps** [comment]
This is what I feared !

It's okay, it will wait :)

**Comment by Freeeaky** [comment]
You mentioned the wiki, do you want an account there? I can create one for you.

**Comment by vincentducorps** [comment]
Yes I want. I wanted to suggest you because I have already notice errors on the wiki :)

Use Deewarz as nickname please :)

**Comment by Freeeaky** [comment]
Created, could you join IRC so I can send you your password ?
Or give me your E-Mail ?

**Comment by vincentducorps** [comment]
I send you a PM on GTA Forums :)

**Comment by Freeeaky** [comment]
Okay, http://gtaforums.com/user/865190-freakyy/

**Comment by vincentducorps** [comment]
Can we send a PM on the forum, I have not found button?

**Comment by Freeeaky** [comment]
http://gtaforums.com/index.php?app=members&module=messaging&section=send&do=form&entered_name=Freakyy

**Comment by vincentducorps** [comment]
Sorry, you don't have permission for that!
[#10222] You are not allowed to use the private messaging system.

I'm looking at the documentation but I not find how to activate ...

**Comment by vincentducorps** [comment]
I send you an email on your Gmail address :)

**Comment by Freeeaky** [comment]
I've been working on this for GTA:MP and it's a pain in the ass. I'll probabbly add it tho once I succeed :)

**Comment by vincentducorps** [comment]
Awesome !
